src/experimentation-opportunities/experimentation-opportunities.js

```diff
@@ -11,17 +11,251 @@
+    return;
+  }
+  // eslint-disable-next-line no-param-reassign
+  oppty.recommendations = getRecommendations(lambdaResult);
+}
+
+async function processHighOrganicLowCtrOpportunities(opportunites, context, site) {
+  const { sqs, log } = context;
+  const highOrganicLowCtrOpportunities = opportunites.filter((oppty) => oppty.type === 'high-organic-low-ctr')
+    .map((oppty) => {
+      const { pageViews, trackedPageKPIValue, trackedKPISiteAverage } = oppty;
+      const opportunityImpact = Math.floor(pageViews
+          * (trackedKPISiteAverage - CTR_THRESHOLD_MARGIN - trackedPageKPIValue));
+      return {
+        ...oppty,
+        opportunityImpact,
+      };
+    });
+  log.info(`Found ${highOrganicLowCtrOpportunities.length} high organic low CTR opportunities`);
+  highOrganicLowCtrOpportunities.sort((a, b) => b.opportunityImpact - a.opportunityImpact);
+  const topHighOrganicLowCtrOpportunities = highOrganicLowCtrOpportunities.slice(
+    0,
+    MAX_OPPORTUNITIES,
+  );
+  const topHighOrganicUrls = topHighOrganicLowCtrOpportunities.map((oppty) => ({
+    url: oppty.page,
+  }));
+  log.info(`Triggering scrape for [${JSON.stringify(topHighOrganicUrls, null, 2)}]`);
+  await sqs.sendMessage(process.env.SCRAPING_JOBS_QUEUE_URL, {
+    processingType: 'default',
+    jobId: site.getId(),
+    urls: topHighOrganicUrls,
+  });
+  // wait for the scrape to complete
+  // TODO: replace this with a SQS message to run another handler to process the scraped content,
+  // to eliminate duplicate lambda run time
+
+  // generate the guidance for the top opportunities in parallel
+  const promises = topHighOrganicLowCtrOpportunities.map(
+    (oppty) => updateRecommendations(oppty, context, site),
+  );
+  await Promise.all(promises);
+  for (const oppty of topHighOrganicLowCtrOpportunities) {
+    // eslint-disable-next-line no-await-in-loop
+    // await updateRecommendations(oppty, context, site);
+    // update the oppty in the opporrtunities list
+    const index = opportunites.findIndex(
+      (opp) => opp.page === oppty.page && opp.type === oppty.type,
+    );
+    if (index !== -1) {
+      // eslint-disable-next-line no-param-reassign
+      opportunites[index] = oppty;
+    }
+  }
+}
+
+async function createOrUpdateOpportunityEntity(opportunity, context, existingOpportunities) {
+  const { log, dataAccess } = context;
+  const { Opportunity } = dataAccess;
+  const existingOpportunity = existingOpportunities.find(
+    (oppty) => (oppty.getType() === opportunity.type) && oppty.getData()
+    && (oppty.getData().page === opportunity.data.page),
+  );
+  if (existingOpportunity) {
+    if (existingOpportunity.getStatus() === 'NEW') {
+      // remove and create a new opportunity entity with new data
+      log.info(`[${opportunity.type}] Opportunity entity with status: NEW for ${opportunity.data.page} exists, so removing it and creating a new opportunity entity`);
+      await existingOpportunity.remove();
```

Comment:
@habansal 
>That is getting lost when a new audit is run and finds the same opportunity type for a page.

yes, that's expected. For demo purpose, if you want to override any property, you can 
- disable the audit worker for that site, that way we won't accidentally change any opportunity
- change the status to `INPROGRESS` or `PATCHED/MODIFIED` or something like that

Reply:
@rpapani Not really.
See the decision that was taken here in terms of product requirement:
https://cq-dev.slack.com/archives/C05A45JBP9N/p1732545991035499

Reply:
@habansal I think its better to use a different `origin` for backoffice entries, so that the audit will ignore those entries and can delete the opportunities that are not coming in the latest audit, if the status is NEW. I've added this feedback to slack thread, once we decide, I'll update the code accordingly.